src/core/buildAndValidateConfig.js

```diff
@@ -31,14 +27,10 @@ export default ({
   });
   config.validate();
   setErrorsEnabled(config.errorsEnabled);
-  const parsedQueryString = queryString.parse(window.location.search);
-  logCommand({
-    enabled:
-      parsedQueryString[logQueryParam] !== undefined
-        ? stringToBoolean(parsedQueryString[logQueryParam])
-        : config.logEnabled
-  });
+  setLogEnabled(config.logEnabled, { persist: false, highPriority: false });
```

Comment:
Setting persist and highPriority feels like the LogController is a leaky abstraction.  What if we added a source parameter?  So the sources could be: `"config", "queryString", or "command"` Then the log controller can determine what do to with each source, and how they overwrite each other.

Reply:
Yeah, I think you're right. I'll make the change.